src/gluonts/model/deepar/_estimator.py

```diff
@@ -401,7 +403,8 @@ def create_training_data_loader(
         **kwargs,
     ) -> DataLoader:
         input_names = get_hybrid_forward_input_names(DeepARTrainingNetwork)
-        instance_splitter = self._create_instance_splitter("training")
+        with env._let(max_idle_transforms=maybe_len(data) or 0):
```

Comment:
I’m wondering whether a less invasive way of doing this would be to execute training within this context, instead of the splitter construction: the former could be done in the superclass `GluonEstimator` if I’m not wrong (or by the estimator user when  calling `train`)